scio-bigquery/src/test/scala/com/spotify/scio/bigquery/types/TypeProviderTest.scala

```diff
@@ -450,4 +453,30 @@ class TypeProviderTest extends FlatSpec with Matchers {
     Artisanal1ToTableWithBody.foo(Artisanal1ToTableWithBody(3)) shouldBe 3
   }
 
+
+  class UserDefinedAnnotation1 extends StaticAnnotation
+  class UserDefinedAnnotation2 extends StaticAnnotation
+
+  @UserDefinedAnnotation1
+  @BigQueryType.toTable
+  @UserDefinedAnnotation2
+  case class RecordWithCustomAnnotations(a1: Int)
+
+  it should "support user defined annotations" in {
+    val annotations = typeOf[RecordWithCustomAnnotations].typeSymbol.annotations.map(_.tree.tpe)
+    annotations should contain allOf(
+      typeOf[UserDefinedAnnotation1], typeOf[UserDefinedAnnotation2])
+  }
+
+  @UserDefinedAnnotation1
+  @BigQueryType.fromSchema(
+    """{"fields": [ {"mode": "REQUIRED", "name": "f1", "type": "INTEGER"} ]}""")
+  @UserDefinedAnnotation2
+  class ClassWithCustomAnnotations
+
```

Comment:
do you think it's worthwhile testing with other macro annotations that change the AST or is this sufficient